applications/dashboard/models/class.mediamodel.php

```diff
@@ -218,6 +218,10 @@ public function saveUploadedFile(UploadedFile $file, array $extraArgs = []): arr
             'Size' => $file->getSize(),
         ]);
 
+        if (in_array('bypassUploadLimits', $extraArgs)) {
```

Comment:
This should be wrong. The `in_array()` functions looks at values not keys. What I would rather see is a PHP best practice where you declare defaults on `$extraArgs` as the first line of the method like this:

```php
$extraArgs += [
    'bypassUploadLimits' => false,
];
```

This shows other developers what can be in `$extraArgs` and also allows you to use the values below without worrying whether or not they exist. Then you call is a simple call to a setter. Also good for code coverage in tests as you don't have another code path with the if statement.